### PR TITLE
feat: APP-305 update projects page color

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: APP-398

- update projects page colors for terrasos
- hide irrelevant components
- implement and use new footer

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
This looks great. Just testing a bit for now.

image

Not directly related to projects page but we need to update the button color in the cookies banner. That also made me wonder, do we actually want to link to our own privacy policy and  terms of service on the terrasos app? cc/ @erikalogie @clevinson @S4mmyb 

image

I think all terrasos project will have an image but just in case, do we have a version of our default project image using terrasos colors? @erikalogie 

image

Should we add to a link to our website on this regen network logo? @erikalogie

**Comment by flagrede** [comment]
@erikalogie Here's the projects page with updated colors for terrasos: https://deploy-preview-2504--regen-marketplace.netlify.app/projects/1

**Comment by erikalogie** [comment]
> This looks great. Just testing a bit for now.
> 
> image
> 
> Not directly related to projects page but we need to update the button color in the cookies banner. That also made me wonder, do we actually want to link to our own privacy policy and terms of service on the terrasos app? cc/ @erikalogie @clevinson @S4mmyb

@flagrede we decided in the core design call today to just get rid of this cookie banner for now.
> 
> image
> 
> I think all terrasos project will have an image but just in case, do we have a version of our default project image using terrasos colors? @erikalogie

We could use this one: https://www.figma.com/design/MuSpCtCdU2ns4cFAsPfvsx/Text-Styles-%26-Components?node-id=9569-27275&t=3JruLwgzJxRyy5Y7-1
> 
> image
> 
> Should we add to a link to our website on this regen network logo? @erikalogie

Yes, that would be great!

**Comment by erikalogie** [comment]
> @erikalogie Here's the projects page with updated colors for terrasos: https://deploy-preview-2504--regen-marketplace.netlify.app/projects/1

This looks totally great for now! Should we have a follow-up task to fully implement the design system, complete with dark mode, post-COP?

**Comment by wgardiner** [comment]
The result looks good, but I'm confused by the styling approach. It looks like rather than taking advantage of the Tailwind semantic design token variables from `tailwind.common.js` `theme.extend.colors.sc` it looks like in general you're checking which marketplace client is active and then using conditionals to apply MUI variables. Is there a reason you took this approach? In the interest of timing maybe this is okay, but it means that we'll have to add additional conditionals for each additional client we want to support instead of just referencing the design token which has variables set up for all clients.

File: web-components/src/components/buttons/OutlinedButton.tsx [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+interface OutlinedButtonProps extends ButtonProps {
+  colorScheme?: ColorScheme;
+}
+
+const OutlinedButton = styled(Button)<OutlinedButtonProps>(
+  ({ theme, colorScheme = 'regen' }) => ({
+    color:
+      colorScheme === 'terrasos'
+        ? theme.palette.primary.contrastText
+        : theme.palette.secondary.main,
```
Comment (wgardiner) [inline web-components/src/components/buttons/OutlinedButton.tsx:15]: hard for me to find from the design, but I think something like `rgba(var(--sc-button-text-icon-prefinance) / 1)` or as a class name `text-sc-button-text-icon-prefinance`

File: web-components/src/components/buttons/OutlinedButton.tsx [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
+    backgroundColor: theme.palette.primary.main,
```
Comment (wgardiner) [inline web-components/src/components/buttons/OutlinedButton.tsx:16]: `rgba(var(--sc-button-surface-standard-secondary-default) / 1)` ?

**Comment by flagrede** [comment]
@erikalogie The images in your comment seem to be broken.
I sent you a request for access to the Figma file with the default image we could use for Terrasos.
Also, there's one semantic color that I used, which seemed to be incorrect. Could you tell me if you have the same mapping for this color in Figma: `--sc-button-surface-standard-secondary-hover: var(--ac-neutral-500);` ?

**Review by wgardiner — COMMENTED** [review]
Looks good! Nice work with the design tokens. Is there a good way to switch between Terrasos and Regen client options in the deploy preview?

**Comment by wgardiner** [comment]
Noticed the spacing here is a bit tight on the sold out tag, not sure if this changed in the PR.
image

**Comment by erikalogie** [comment]
> Noticed the spacing here is a bit tight on the sold out tag, not sure if this changed in the PR. image

I noticed this too and forgot to say something!

**Comment by wgardiner** [comment]
Looks like there's an issue with this text too for Regen client
image

**Review by wgardiner — APPROVED** [review]
Just a couple tweaks needed

**Comment by flagrede** [comment]
@wgardiner @erikalogie I was able to fix everything, including the existing spacing issue for the sold-out tag.

**Pull request merged**